website/blog/2023-08-03-Create-an-Apache-Hudi-based-near-real-time-transactional-data lake-using-AWS-DMS-Amazon-Kinesis-AWS-Glue-streaming-ETL-and-data-visualization-using-Amazon-QuickSight.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: "Create an Apache Hudi-based-near-real-time transactional data lake using AWS DMS, Amazon Kinesis, AWS Glue streaming ETL, and data visualization using Amazon QuickSight"
+authors:
+- name: Raj Ramasubbu
+- name: Sundeep Kumar
+- name: Rahul Sonawane 
+category: blog
+image: /assets/images/blog/2023-08-03-Create-an-Apache-Hudi-based-near-real-time-transactional-data-lake-using-AWS-DMS-Amazon-Kinesis-AWS-Glue-streaming-ETL-and-data-visualization-using-Amazon-QuickSight.png
+tags:
+- blog
+- aws
+- aws dms
+- aws kinesis
+- aws glue
+- aws quicksight
+---
```

Comment:
Please stick to the guidelines in the readme - https://github.com/apache/hudi/blob/asf-site/README.md#blogs . For ex. I don't find 'amazon" in the tag. We try to add the domain name in the tags denoting the source of the blog as the last tag. Also, we should highlight the Hudi related components/features.  This blog talks about Change data capture and upserts for near real time ingestions. We should high light those aspects.

Reply:
updated